tests/test_buffer_property.py

```diff
@@ -1,39 +1,41 @@
 "Contains a collection of pytest tests that validates the buffer property."
-import random
-
-import pytest
-
 import nidaqmx
 from nidaqmx.constants import SampleTimingType
 from nidaqmx.errors import DaqError
-from nidaqmx.tests.helpers import generate_random_seed
-from nidaqmx.tests.test_read_write import TestDAQmxIOBase
 
 
-class TestBufferProperty(TestDAQmxIOBase):
-    """Contains a collection of pytest tests that validates the buffer property."""
+def test__buffer__set_int32_property__value_is_set(any_x_series_device):
+    """Test for validating int32 attributes in buffer."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+        task.timing.samp_timing_type = SampleTimingType.SAMPLE_CLOCK
 
-    @pytest.mark.parametrize("seed", [generate_random_seed()])
-    def test_int32_property(self, any_x_series_device, seed):
-        """Test for validating int32 attributes in buffer."""
-        # Reset the pseudorandom number generator with seed.
-        random.seed(seed)
+        # Setting a valid input buffer size of type int32
+        task.in_stream.input_buf_size = 2000000000
+        assert task.in_stream.input_buf_size == 2000000000
 
-        with nidaqmx.Task() as task:
-            task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
-            task.timing.samp_timing_type = SampleTimingType.SAMPLE_CLOCK
-            default_buffer_size = task.in_stream.input_buf_size
 
-            # Setting a valid input buffer size of type int32
-            task.in_stream.input_buf_size = 2000000000
+def test__buffer__set_invalid_int32_value__default_value_is_retained(any_x_series_device):
+    """Test for validating int32 attributes in buffer."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+        task.timing.samp_timing_type = SampleTimingType.SAMPLE_CLOCK
+
+        # Setting a invalid input buffer size greater than int32
+        try:
+            task.in_stream.input_buf_size = 4000000000
+        except DaqError:
             assert task.in_stream.input_buf_size == 2000000000
```

Comment:
Oops, there's a better way to do this:

```suggestion
        # Setting a invalid input buffer size greater than int32
        with pytest.raises(DaqError):
            task.in_stream.input_buf_size = 4000000000

        assert task.in_stream.input_buf_size == 2000000000
```